pkg/controller/gslb/helpers.go

```diff
@@ -1,25 +1,12 @@
 package gslb
 
 import (
-	"encoding/json"
-
 	ohmyglbv1beta1 "github.com/AbsaOSS/ohmyglb/pkg/apis/ohmyglb/v1beta1"
-	yamlConv "github.com/ghodss/yaml"
+	"github.com/AbsaOSS/ohmyglb/pkg/controller/gslb/internal/utils"
 )
 
 //YamlToGslb takes yaml and returns Gslb object
 func YamlToGslb(yaml []byte) (*ohmyglbv1beta1.Gslb, error) {
-	// yamlBytes contains a []byte of my yaml job spec
-	// convert the yaml to json
-	jsonBytes, err := yamlConv.YAMLToJSON(yaml)
-	if err != nil {
-		return &ohmyglbv1beta1.Gslb{}, err
-	}
-	// unmarshal the json into the kube struct
-	var gslb = &ohmyglbv1beta1.Gslb{}
-	err = json.Unmarshal(jsonBytes, &gslb)
-	if err != nil {
-		return &ohmyglbv1beta1.Gslb{}, err
-	}
-	return gslb, nil
+	//TODO: point to discussion: consider common utils package
+	return utils.YamlToGslb(yaml)
```

Comment:
Looks like whole `helpers.go` can be removed now

Reply:
Hi, I'd like to but utils is now internal and is visible only from gslb package. 
Thats why I had to wrote wrapper function because it is used e2e tests and they cannot see internal utils. 
This is only one point which is a bit wierd. 
So we can 

- keep it as it is for now (gslb exposes controller + YamlToGslb)
- or move e2e next to the gslb_controller  
- or make utils as package on global level
- create repo github.com/AbsaOSS/utils - having some common utils for  Absa go projects
- maybe there is better idea :)

Reply:
i believe e2e tests under `pkg/test/` and associated make target can be just removed. It was attempt to create e2e test flow with mechanics provided by operator framework but it appeared to be pretty limited. And after introduction of terratest we don't really need them anymore and it is kind of dead code nobody actually uses. So i believe it is a perfect chance to get rid of them